### ISSUE TITLE
Add way to set renamed workspace

### ISSUE THREAD

**Issue by thomascrha** [mikkasendke/sessionizer.wezterm#8]
This isn't really a bug but a feature request. What you be nice is when sessionizer is creating the workspace from the search list it wouldn't use the entire path but rather the basename i.e. `/path/to/myproject` would just be `myproject`. This could be a config option - maybe `use_path_basename=false` - and if `true` use the basename.

**Comment by mikkasendke** [comment]
Yes that is true, I have another branch, that would make this really easy and I will have a built-in for that. It works different in terms of configuration from main though and I still have to finish rewriting the readme (that's why it isn't merged yet).

**Comment by mikkasendke** [comment]
well maybe not a built-in idk it is just one line for a user in the new version anyway so yeah maybe an example in the readme

**Comment by thomascrha** [comment]
Sorry if that came off as a demand - its not - what ever you think is best to invoke this functionality - if its a config var or just some - or a one line in the readme - thanks for your quick reply - im going to have a play with this new branch

**Comment by mikkasendke** [comment]
No worries, my response was not meant to be hostile in the slightest or anything, maybe I was a bit direct. I think this is a genuine concern and having good and easy ways of changing how the entries in the list are shown is something multiple people would benefit from. It's just that the new branch in which I somewhat address this completely breaks compatibility and the readme isn't finished and it maybe still needs at least some polish. Thank you for submitting the issue, it is always appreciated :)

**Comment by mikkasendke** [comment]
@thomascrha the new version with breaking changes in terms of configuration is out now but this is now easily possible by using processing functions
